datadog/Tiltfile

```diff
@@ -105,22 +132,36 @@ def datadog_keys_secret():
     )
 
 
-def helm_resources():
+def helm_resources(agent_image_tag = None):
+    """Create Tilt resources required to install Datadog Agent from Helm charts.
+
+    Args:
+      agent_image_tag: Sets a specific image tag version for the Datadog Agent.
+    """
+    release_name = "datadog-agent"
     helm_repo(
         "helm-datadog-repo",
         "https://helm.datadoghq.com",
         labels = ["datadog"],
     )
 
     chart_values = os.path.join(extension_path, "values.yaml")
+    flags = ["--values", chart_values]
+
+    if agent_image_tag:
+        flags.extend(["--set", "agents.image.tag=%s" % agent_image_tag])
+        flags.extend(["--set", "clusterAgent.image.tag=%s" % agent_image_tag])
+        flags.extend(["--set", "clusterChecksRunner.image.tag=%s" % agent_image_tag])
+
+        release_name = "%s-%s" % (release_name, agent_image_tag.replace(".", "-"))
```

Comment:
When using a long tag such as [`6.50.3-fix-missing-repo-digest`](https://hub.docker.com/layers/datadog/agent/6.50.3-fix-missing-repo-digest/images/sha256-e26359b5169f92ac5861dbb54460321c640ea254c199ad86c8cd2e7d1b4184ea?context=explore), the Helm install process fails with this error:

```
Running cmd: ['helm', 'upgrade', '--install', '--values', '/Users/noelle/dev/DirectEmployers/tilt-extensions/datadog/values.yaml', '--set', 'agents.image.tag=6.50.3-fix-missing-repo-digest', '--set', 'clusterAgent.image.tag=6.50.3-fix-missing-repo-digest', '--set', 'clusterChecksRunner.image.tag=6.50.3-fix-missing-repo-digest', 'datadog-agent-6-50-3-fix-missing-repo-digest', 'helm-datadog-repo/datadog']
Release "datadog-agent-6-50-3-fix-missing-repo-digest" does not exist. Installing it now.
    Error: 1 error occurred:
        * Service "datadog-agent-6-50-3-fix-missing-repo-digest-cluster-agent-admission-controller" is invalid: metadata.name: Invalid value: "datadog-agent-6-50-3-fix-missing-repo-digest-cluster-agent-admission-controller": must be no more than 63 characters
```

Reply:
I'll what I can do about it

Reply:
All fixed :)